tests/db_migrator_input/minigraph.xml

```diff
@@ -19,9 +19,17 @@
     </Devices>
    </PngDec>
   <MetadataDeclaration>
-    <Devices xmlns:a="http://schemas">
+    <Devices xmlns:a="http://schemas.datacontract.org/2004/07/Microsoft.Search.Autopilot.Evolution">
       <a:DeviceMetadata>
-        <a:Properties/>
+        <a:Name>SONiC-Dummy</a:Name>
+        <a:Properties>
+          <a:DeviceProperty>
+            <a:Name>DnsNameserverResources</a:Name>
+            <a:Value>
+             1.1.1.1;8.8.8.8
```

Comment:
>1.1.1.1;8.8.8.8 [](http://example.com/codeflow?start=13&length=15)

Did you test empty Value?

Reply:
Actually, we should revert minigraph schema code change and test cases.

Reply:
https://github.com/sonic-net/sonic-buildimage/pull/15727